src/EoS/Nuclear/NuclearEoS.cpp

```diff
@@ -109,11 +109,7 @@ void findtemp_NR_bisection( const real lr, const real lt_IG, const real ye, cons
 //                                     120 : temp too high
 //                                     121 : temp too low
 //                                     122 : temp NaN
-//                                     130 : entr too high
-//                                     131 : entr too low
 //                                     132 : entr NaN
-//                                     140 : pres too high
-//                                     141 : pres too low
```

Comment:
Since we retains the error code now (although they are not returned), should we keep this information?

Reply:
I added back the error codes comments.